src/include/storage/storage_defs.h

```diff
@@ -175,8 +175,12 @@ using ColumnMap = std::unordered_map<catalog::col_oid_t, col_id_t>;
 /**
  * Used by execution and storage layers to map between col_oids and offsets within a ProjectedRow
  */
-using InverseColumnMap = std::unordered_map<col_id_t, catalog::col_oid_t>;
 using ProjectionMap = std::unordered_map<catalog::col_oid_t, uint16_t>;
+using InverseColumnMap = std::unordered_map<col_id_t, catalog::col_oid_t>;
+/**
+ * Used by SqlTable to map between col_oids in Schema and their {default_value, attribute_size}
+ */
+using DefaultValueMap = std::unordered_map<catalog::col_oid_t, std::pair<byte *, uint8_t>>;
```

Comment:
We should probably use a concurrent unordered map here.  Even though adding is single-threaded there are still potential lookups happening concurrently with that operation.

Reply:
You are right. I've changed it to common::ConcurrentMap